go/libraries/doltcore/cherry_pick/cherry_pick.go

```diff
@@ -299,6 +357,20 @@ func cherryPick(ctx *sql.Context, dSess *dsess.DoltSession, roots doltdb.Roots,
 	return result, cherryCommitMeta.Description, nil
 }
 
+func noChangesBetweenRoots(root1, root2 doltdb.RootValue) (bool, error) {
```

Comment:
Is `rootsEqual()` a better name? Maybe a method on RootValue is needed? Seems odd that you need to go to all this effort to check that roots are equal

Reply:
Changed in 3ee48c2eed346a5864307e6a4aa2d7e1340b36c7